### PR TITLE
Fixed broken GIBCT test

### PR DESCRIPTION
**Pull request by U-DON**

### CODE REVIEW

File: src/js/gi/reducers/index.js [hunk 1]
```diff
@@ -9,7 +11,7 @@ import InstitutionDetailsFilterReducer from './filter';
 import ProfileReducer from './profile';
 import CalculatorReducer from './calculator';
 
-const rootReducer = {
+const rootReducer = combineReducers({
```
Comment (jbalboni) [inline src/js/gi/reducers/index.js]: Does the app work with this? I think the common store call will fail.

Comment (U-DON) [inline src/js/gi/reducers/index.js]: Oh, I see now that the common store is doing the `combineReducers` already. So it looks like I'll have to do this within the tests themselves.

Comment (U-DON) [inline src/js/gi/reducers/index.js]: In which case, probably using `createCommonStore` again in the tests would avoid the verbosity of using `createStore` and `combineReducers` in each of them.

Comment (jbalboni) [inline src/js/gi/reducers/index.js]: Yeah. I think that's what I did in the other tests

**Review by jbalboni — APPROVED** [review]

**Review by bshyong — APPROVED** [review]
LGTM

**Pull request merged**